query/tests/it/servers/http/http_query_handlers.rs

```diff
@@ -128,7 +136,7 @@ async fn test_async() -> Result<()> {
         assert!(result.next_uri.is_none());
         assert!(result.schema.is_none());
```

Comment:
> assert!(result.schema.is_none()); 

change is_none to is_some()

Reply:
before this change, only the first response contains schema.

now every response may contain or not contain schema,  only depends on if data is empty.

Reply:
fixed, thank you for remind